ts/handler_utils/distributed/deepspeed.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import os
+
+import deepspeed
+
+from ts.context import Context
+
+
+def get_ds_engine(model, ctx: Context):
+    model_dir = ctx.system_properties.get("model_dir")
+    ds_config = None
+    checkpoint = None
+    if "deepspeed" in ctx.model_yaml_config:
+        # config: the deepspeed config json file path.
+        # deepspeed config parameters:
+        # https://github.com/microsoft/DeepSpeed/blob/master/deepspeed/inference/config.py
+        if "config" in ctx.model_yaml_config["deepspeed"]:
+            ds_config = os.path.join(
+                model_dir, ctx.model_yaml_config["deepspeed"]["config"]
+            )
+            if not os.path.exists(ds_config):
+                raise ValueError(
+                    f"{ctx.model_name} has no deepspeed config file {ds_config}"
```

Comment:
does deepspeed have some configuration object we can use directly, there's a lot of config files now

Reply:
Deepspeed has tons of configuration. it is difficult to cover all of them. A central configuration file is the simplest and easiest way to maintain.